framework/web/CHttpCookie.php

```diff
@@ -48,6 +48,11 @@ class CHttpCookie extends CComponent
 	 * Note, this property is only effective for PHP 5.2.0 or above.
 	 */
 	public $httpOnly=false;
+    /**
+     * @var array Additional setcookie options like "SameSite".
+     * This property only works for PHP 7.3.0 or above.
+     */
+	public $options=array();
```

Comment:
It seems there are no additional options other than SameSite. How about handing it the same way it's done in Yii 2: https://github.com/yiisoft/yii2/blob/master/framework/web/Cookie.php#L80 and Yii 3: https://github.com/yiisoft/yii-web/blob/master/src/Cookie.php#L79

Reply:
> there are no additional options other than SameSite

... yet

I think that we want the code to be flexible for the future and prevent more pull requests for every other flag that browser vendors want to add.

Reply:
That complicates the syntax. Additional properties could be added as needed.

Reply:
> That complicates the syntax

That is a bold claim, but hey... have it your way.
I changed it.